inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/utils/HttpManager.java

```diff
@@ -218,4 +226,13 @@ private HttpGet getHttpGet(String url) {
         return new HttpGet(url);
     }
 
+    public Map<String, String> getAuthHeader() {
+        Map<String, String> header = new HashMap<>();
+        try {
+            header.put(BasicAuth.BASIC_AUTH_HEADER, BasicAuth.genBasicAuthCredential(secretId, secretKey));
+        } catch (Exception e) {
+            LOGGER.error("getAuthHeader error", e);
```

Comment:
please make the log content more readable.

Reply:
ok